mllib/src/main/scala/org/apache/spark/mllib/stat/Statistics.scala

```diff
@@ -158,4 +158,30 @@ object Statistics {
   def chiSqTest(data: RDD[LabeledPoint]): Array[ChiSqTestResult] = {
     ChiSqTest.chiSquaredFeatures(data)
   }
+
+  /**
+   * Conduct a 1-sample Anderson-Darling test for the null hypothesis that the data
+   * comes from a given theoretical distribution. The Anderson-Darling test is an alternative
+   * to the Kolmogorov-Smirnov test, and is more adequate at identifying departures from the
+   * theoretical distribution at the tails. The implementation returns an `ADTestResult`, which
+   * includes the AD statistic, the critical values at varying significance levels, and
+   * the null hypothesis. Note that the critical values are calculated assuming the parameters
+   * have been calculated from the data sample. If the parameters for the theoretical distribution
+   * are not in a valid domain, throws an exception.
+   * @param data `RDD[Double]` the data to be test
```

Comment:
Ah I see. I was going off of what I saw in Statistics.chiSqTest that seemed to note in some occasions. I'll remove. Thanks!